api/v3/Event/Generatewebinarattendance.php

```diff
@@ -46,30 +46,31 @@ function civicrm_api3_event_generatewebinarattendance($params) {
 		throw new \Civi\API\Exception\UnauthorizedException('Invalid verification token.');
 	}
 
-
-	$key = $_ENV['ZOOM_API_SECRET'];
+	$settings = CRM_NcnCiviZoom_Utils::getZoomSettings();
+	$key = $settings['api_key'];
```

Comment:
same as above, using API key where it expecting secret key and using secret key where its expecting API key. Please fix this.